Source/Posts/2017/07-Introduce-data-conversion-naming.xyl

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xyl-overlay href="hoa://Application/Overlays/Article.xyl"?>
+<?xyl-meta name="title" value="Introduce data conversion naming"?>
+<?xyl-meta name="date" value="2017-07-01T07:54:50+02:00"?>
+
+<overlay xmlns="http://hoa-project.net/xyl/xylophone">
+  <article id="main">
+    <p>
+      For 2017, we <a href="http://discourse.hoa-project.net/t/roadmap-to-2017-a-draft/235">defined
+      a roadmap</a> composed of Request For Comments (RFC) that are discussed
+      and implemented. One if these RFC is about
+      <a href="https://github.com/hoaproject/Central/issues/54">naming convention
+      and simplification</a>.
+    </p>
+    <p>
+      So far, we use this formalism: <code>getFoo()</code> to name a method
+      that returns the value <code>foo</code>. This can be a direct attribute, or a
+      computation. To get this data within another form, i.e. to convert this
+      data into another type, we don't have any formalism yet. For instance,
+      if <code>foo</code> is an array, and we would like to get it as a string,
+      we will probably name a method like <code>getFooAsString()</code> but this
+      is not deterministic.
+    </p>
+    <h2>Property access</h2>
+    <p>
+      Initially we decided to remove the <code>get</code> prefix from getters
+      methods because it's not mandatory to understand the code. Also in many
+      languages this prefix have been drop/is not present so we can ask if it's
+      still relevant.
+    </p>
+    <p>
+      After receiving some <a href="https://github.com/hoaproject/Central/issues/54#issuecomment-281268963">community
+      feedbacks</a> it seems that this prefix is too present in the PHP world.
+      Removing it will make code more complex to read and understand for PHP
```

Comment:
Not sure here... The code will be less consistent without the prefix but it's more complex to read...

Maybe we can use : 

"Remove it will make the code harder to read and understand ..." ?